### ISSUE TITLE
Add Code of Conduct

### ISSUE THREAD

**Issue by KiranBaliga** [jahnvisahni31/DesignDeck#1]
I would like to add a code of conduct developed for your repository. Please let me know if it is okay to proceed. Please assign to me and add GSSoC24 Ext and level1 Labels.

**Comment by jahnvisahni31** [comment]
yeah you could go ahead with it and also make changes in readme.md according to that

**Comment by jahnvisahni31** [comment]
not valid issue
